### DISCUSSION TITLE
Feature request: getPrevUrls(2) and getNextUrls(2)

### DISCUSSION THREAD

**Discussion by shoored** [craftcms/element-api#151] [Ideas]
When working with ElementAPI and a Javascript front-end — for example Vue — the pagination options are a bit slim.

It would be nice to be able to create the same pagination links as we can with Twig Pagination while also preserving the querystrings as is with the current prev and next links in the JSON meta object.

**Comment by brandonkelly** [comment]
Can you elaborate a bit on this?

**Comment by simondotwhite** [comment]
I think @studiosyntax means the ability to return the URLs for multiple previous/next links. 

https://docs.craftcms.com/v3/dev/tags/paginate.html#nearby-page-links

That's the documentation link for it.

**Comment by bkmorse** [comment]
This would be an awesome addition to this plugin.

**Comment by furioursus** [comment]
Another +1 here. Can't always use GraphQL for projects, but Element API still gets us through a lot!
